ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -149,6 +149,11 @@ host_monitoring_cron:
   minute: "*/5"
   job: ops-runner -f -s 15 -n csnm.network.int /usr/bin/cron-send-network-metrics
 
+- name: report rpm versions daily
+  hour: "1"
+  minute: "0"
+  job: ops-runner -f -s 120 -n csdov.rpm.versions /usr/bin/cron-send-docker-oc-versions
```

Comment:
What would happen on zabbix if the item isn't defined? Will it just be ignored?

I'm not sure we need it on zabbix. It's not a metric. We are simply using hawkular as a medium to transfer data that we can check against expected values in manageIQ policies.